nfts/frontend/src/Components/NFT/TransferNFT.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import TransferNFTService from "../../utils/transferNft";
+import "./TransferNFT.css";
+
+export const TransferNFT = ({ sdk }) => {
+  const [tokenId, setTokenId] = useState("");
+  const [to, setTo] = useState("");
+  const [from, setFrom] = useState("");
+  const [result, setResult] = useState(null);
+  const [errors, setErrors] = useState(null);
+
+  let handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const result = await TransferNFTService(tokenId, to, from);
+      if (result.status == "USER_CONSENT_NEEDED") {
+        const consentToken = result.consentToken;
+        const consent = await sdk.getConsent(consentToken);
+        if (consent.status == "QUEUED") {
+          setResult(consent);
+          console.log(result);
+        } else {
+          setErrors(consent.status);
```

Comment:
looks like the error doesn't appear anywhere on the UI

Reply:
it will be presented as an alert on UI